src/features/auth/pages/signup/SignupConfirmationEmailSent/EmailAttemptsLeft.stories.tsx

```diff
@@ -22,8 +22,12 @@ const variantConfig: Variants<typeof EmailAttemptsLeft> = [
   },
 ]
 
-const Template: VariantsStory<typeof EmailAttemptsLeft> = () => (
-  <VariantsTemplate variants={variantConfig} Component={EmailAttemptsLeft} />
+const Template: VariantsStory<typeof EmailAttemptsLeft> = (args) => (
+  <VariantsTemplate
+    variants={variantConfig}
+    Component={EmailAttemptsLeft}
+    defaultProps={{ ...args }}
```

Comment:
```suggestion
    defaultProps={ args }
```
C'est nécessaire de spread ?

Reply:
dans le message de commit et la description de cette PR, j'ai marqué oui, c'est nécessaire

je ne comprends pas

sans, il y a une 60aine de fichiers qui passent, et une 30aine qui ont une erreur

le seul moyen que j'ai trouvé c'est de spread

si quelqu'un a une meilleure solution, ça serait avec plaisir

Reply:
Pardon j'ai du zapper l'info ! Pas de meilleur solution 👍